CONTRIBUTING.md

```diff
@@ -118,6 +118,29 @@ Your topic will also contain links to the sample. Link directly to the sample's
 
 For more information, see the [Samples Readme](https://github.com/dotnet/docs/blob/master/samples/README.md).
 
+## The C# interactive experience #
+
+Short code samples in C# can use the `csharp-interactive` language tag to
+specify a C# sample that run in the browswer. When these sample are built, the code sample
```

Comment:
run --> runs
When these sample are built, the code sample --> When such a sample is built, it